README.md

```diff
@@ -497,6 +497,16 @@ The `search-picker` form field is an input field with search functionality which
         - The success message is displayed to the user either as a flash message or through a browser event. The display method depends on the value of the `$hasSession` property, which is true by default.
     * Another way to customize the success message is to override the `successMessage()` method in the component class.
 
+* `protected function canSubmit(): bool`
+    * Purpose: Check if the user is authorized to submit the form
+
+* `protected function isFieldRequired($fieldKey): bool`
```

Comment:
I think this should also describe the default behaviour for all functions. Since `extraCheckRequiredField` is included in the default implementation of `isFieldRequired`, should really both be open for being overwritten? I think the extra check should be sufficient to allow customisation.

Regarding the required logic it should also be mentioned that this only affects the `*` indicator in the label, so does not apply to actual validation and form fields without a label (such as cards)